csvdata/csvdata.go

```diff
@@ -0,0 +1,76 @@
+package csvdata
+
+import (
+	"encoding/csv"
+	"errors"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"time"
+
+	"github.com/Diegoplas/2022Q2GO-Bootcamp/config"
+	"github.com/Diegoplas/2022Q2GO-Bootcamp/model"
+	"github.com/wcharczuk/go-chart"
```

Comment:
I recommend to separate the import libraries like

import (
// standar libraries

// our libraries

// third party libraries)

Just skipping one empty row from each section